crates/apub/src/objects/comment.rs

```diff
@@ -184,6 +184,7 @@ impl ApubObject for ApubComment {
       updated: note.updated.map(|u| u.naive_local()),
       deleted: None,
       ap_id: Some(note.id.into()),
+      distinguished: None, // TODO how to do this?
```

Comment:
Can you think of any field we could use for this? Or just ignore for now.

Reply:
I dont see any relevant field in activitystreams. You can just call it distinguished, and later we can migrate to a different name. You will have to add the field in assets/lemmy/context.json, and to assets/lemmy/objects/note.json.

Keep in mind that we still dont federate admin status, so we cant verify that this was really sent by a mod/admin.

Reply:
Okay I think I got those now.